ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -524,64 +524,109 @@ static odla_value binary_eltwise_s32(dnnl::algorithm alg, dnnl::memory lhs_mem,
   return v;
 }
 
+void expand_dims(odla_value& from, odla_value& to) {
+  // from_shape is [1, 5], to_shape is [1,4,1], we expand from_shape to [1,1,5]
```

Comment:
do you mean "to_shape" is [1,5,1] ?

Reply:
should we declare  "expand_dims", "broadcast_op" and "broadcast_func" as static?

Reply:
Otherwise, looks good to me.